code/modules/mob/living/carbon/human/death.dm

```diff
@@ -23,14 +23,13 @@
 			E.droplimb(1, 1)
 	dropBorers()
 	var/gib_radius = 0
-	if(reagents.has_reagent(LUBE))
+	if(reagents?.has_reagent(LUBE))
```

Comment:
`reagents` is not supposed to be null, so this is masking another bug

Reply:
fair enough, will revert this